roles/dns/manage-dns-zones/tasks/named/process-views.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Ensure the final view directory exists
+  file:
+    path: "{{ dns_zone_temp_config_dir }}/view"
+    state: directory
+
+- include_tasks: process-zones.yml
+  with_items:
+    -  "{{ dns_data.views }}"
+  loop_control:
+    loop_var: "view"
+
+- name: Assemble the final view configuration
```

Comment:
I see that we restart named two times in a row. Is this something that we could wrap in a block and then notify? I haven't tried this before, so I'm not sure if this would work, but just didn't know if it was better to just restart named once instead of twice.

Reply:
the purpose of `notify` is to only do it once - at the end of the execution... hence, this is correct as-is